databaseAnnotator/src/main/java/org/cbioportal/database/annotator/AnnotateRecordsWriter.java

```diff
@@ -0,0 +1,130 @@
+                 for (MutationEvent event : mutationEvents) {
+                     if (event.getPROTEIN_CHANGE().equals(annotatedEvent.getPROTEIN_CHANGE())) {
+                         updateMutationsInDb(annotatedEvent, event);
+                         deleteMutationEvent(annotatedEvent);
+                     }
+                 }
+            }
+        }
+        else {
+            log.error("Event " + annotatedEvent.getMUTATION_EVENT_ID() + " unable to be fixed.");
+        }
+    }
+
+    private List<MutationEvent> getDuplicatedMutationEvents(MutationEvent annotatedEvent) {
+        MutationEvent qMutationEvent = alias(MutationEvent.class, BatchConfiguration.MUTATION_EVENT_TABLE);
+        List<MutationEvent> mutationEvents = databaseAnnotatorQueryFactory.select(
+                    Projections.constructor(MutationEvent.class, $(qMutationEvent.getMUTATION_EVENT_ID()),
+                            $(qMutationEvent.getCHR()),
+                            $(qMutationEvent.getSTART_POSITION()),
+                            $(qMutationEvent.getEND_POSITION()),
+                            $(qMutationEvent.getREFERENCE_ALLELE()),
+                            $(qMutationEvent.getTUMOR_SEQ_ALLELE()),
+                            $(qMutationEvent.getPROTEIN_CHANGE()),
+                            $(qMutationEvent.getMUTATION_TYPE()),
+                            $(qMutationEvent.getONCOTATOR_CODON_CHANGE()),
+                            $(qMutationEvent.getONCOTATOR_PROTEIN_POS_START()),
+                            $(qMutationEvent.getONCOTATOR_PROTEIN_POS_END())))
+                    .from($(qMutationEvent))
+                    .where($(qMutationEvent.getCHR())
+                                .eq(annotatedEvent.getCHR())
+                            .and($(qMutationEvent.getSTART_POSITION())
+                                .eq(annotatedEvent.getSTART_POSITION()))
+                            .and($(qMutationEvent.getEND_POSITION())
+                                .eq(annotatedEvent.getEND_POSITION()))
+                            .and($(qMutationEvent.getTUMOR_SEQ_ALLELE())
+                                .eq(annotatedEvent.getTUMOR_SEQ_ALLELE()))
+                            .and($(qMutationEvent.getMUTATION_TYPE()).eq(annotatedEvent.getMUTATION_TYPE())))
+                    .fetch();
+        return mutationEvents;
+    }
+
+    private void updateMutationsInDb(MutationEvent eventToBeDeleted, MutationEvent properlyAnnotatedEvent) throws Exception{
+        PreparedStatement pstmt = con.prepareStatement("UPDATE mutation set mutation_event_id = ? where mutation_event_id = ?");
+        pstmt.setInt(1, properlyAnnotatedEvent.getMUTATION_EVENT_ID());
+        pstmt.setInt(2, eventToBeDeleted.getMUTATION_EVENT_ID());
+        pstmt.executeUpdate();
+		log.info("Updating mutations - mutations linked to mutation event " + eventToBeDeleted.getMUTATION_EVENT_ID() + " now linked to " + eventToBeDeleted.getMUTATION_EVENT_ID());
```

Comment:
Logging message describing mutation event id update doesn't make sense here. 

Example of what I saw when I ran the tool: 
`Updating mutations - mutations linked to mutation event 122154 now linked to 122154`


`log.info(... +  " now linked to " + eventToBeDeleted.getMUTATION_EVENT_ID()`  should be 

`log.info(... +  " now linked to " + properlyAnnotatedEvent.getMUTATION_EVENT_ID()`